packages/ia-topnav/src/styles/desktop-subnav.js

```diff
@@ -0,0 +1,43 @@
+import { css } from 'lit-element';
+
+export default css`
+  ul {
+    position: relative;
+    z-index: 2;
+    padding: .8rem 0;
+    margin: 0;
+    font-size: 1.2rem;
+    text-transform: uppercase;
+    text-align: center;
+    background: var(--grey20);
+  }
+
+  li {
+    display: inline-block;
+    padding: 0 15px;
+  }
+
+  a {
+    text-decoration: none;
+    color: var(--subnavLinkColor);
+  }
+
+  a:hover,
+  a:active,
+  a:focus {
+    color: var(--white);
+  }
+
+  .donate svg {
+    width: 16px;
+    height: 16px;
+    vertical-align: -4px;
+    fill: #f00;
+  }
+
+  @media (max-width: 889px) {
+    ul {
+      display: none;
+    }
+  }
```

Comment:
Is this causing the desktop subnav to hide under 890px? If so, I would place this responsibility on its parent component. This change can affect layout in the parent so it should be aware that this is going to happen.